client/app/bundles/App/Components/Core/Alert/Index.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import AlertProps from 'terra-alert/docs/props-table/Alert.md';
+import Markdown from 'terra-markdown';
+import ReadMe from 'terra-alert/docs/README.md';
+import { version } from 'terra-alert/package.json';
+
+// Example Files
+import AlertExample from 'terra-site/lib/examples/alert/AlertExample';
+import ErrorExample from 'terra-site/lib/examples/alert/ErrorExample';
+import WarningExample from 'terra-site/lib/examples/alert/WarningExample';
+import AdvisoryExample from 'terra-site/lib/examples/alert/AdvisoryExample';
+import InfoExample from 'terra-site/lib/examples/alert/InfoExample';
+import SuccessExample from 'terra-site/lib/examples/alert/SuccessExample';
+import CustomExample from 'terra-site/lib/examples/alert/CustomExample';
+import LongTextExample from 'terra-site/lib/examples/alert/LongTextExample';
+import ActionExample from 'terra-site/lib/examples/alert/ActionExample';
+import SmallContainerExample from 'terra-site/lib/examples/alert/SmallContainerExample';
+import DismissibleExample from 'terra-site/lib/examples/alert/DismissibleExample';
+import ActionAndDismissibleExample from 'terra-site/lib/examples/alert/ActionAndDismissibleExample';
+import BuildYourOwnAlertExample from 'terra-site/lib/examples/alert/BuildYourOwnAlertExample';
+
+const AlertExamples = () => (
+  <div>
+    <div id="version">Version: {version}</div>
+    <Markdown id="readme" src={ReadMe} />
+    <Markdown id="props-table" src={AlertProps} />
+    <h2>Alert Examples</h2>
+    <br />
+    <p>
+    PLEASE NOTE: The strings that are internationalized in the following examples are the default titles for each of
+    the alert types (except the custom alert type), and the text for the Dismiss button.
+    </p>
+    <br /><hr /><br />
```

Comment:
There is an [issue logged](https://github.com/cerner/terra-core/issues/382) in terra-core to create a `<hr />` element that would include the ability to set margins on it. This should replace the usage of the `<br/><hr/><br/>` pattern used in  the example. For the time being I've [removed the br elements and styled the hr](https://github.com/cerner/terra-ui.com/pull/51/commits/90670fdd7942086141eea43e0aa583a2900d5499) to match what we'll likely do in the terra-core component.